Gemfile

```diff
@@ -33,8 +33,8 @@ gem 'bootsnap', '>= 1.4.2', require: false
 gem 'simple_form', '~> 4.1'
 gem 'nested_form'
 gem 'figaro', '~> 1.1.1', github: 'laserlemon/figaro'
-gem 'rails_bootstrap_helper', "~> 1.0.1", github: "olimart/rails_bootstrap_helper", branch: "bs3.1"
-gem 'tabs_on_rails', "~> 3.0"
+gem 'rails_bootstrap_helper', '~> 1.0.1', github: 'olimart/rails_bootstrap_helper', branch: 'bs3.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.